### PR TITLE
VZ-5064. Update Customizing OKE LBs doc

### PR DESCRIPTION
**Pull request by michael-cico**
Documents how to override the subnet placement for VZ load balancers for private LB configurations.

### CODE REVIEW

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ Verrazzano sets up the following load balancers on Kubernetes at installation:
 * Load balancer for Istio ingress
 
 Verrazzano allows customizing the load balancers allocated by Oracle Container Engine (OKE) using annotations defined by OKE.
+You can find a detailed discussion of the different load balancer customization annotations in the OKE documentation
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:14]: re-write: For a detailed description of different load balancer customization annotations, see the OKE documentation [here](https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengcreatingloadbalancer.htm).

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 2]
```diff
@@ -61,9 +67,13 @@ spec:
 
 ### Using private IP addresses with a load balancer
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:69]: Using private IP addresses with a load balancer -> Use private IP addresses with a load balancer

**Review by rosemarymarano — COMMENTED** [review]
Few edits

**Review by theabmitra — APPROVED** [review]
lgtm

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md (hunk 1 above)
Comment (michael-cico) [reply content/en/docs/setup/customizing/ociLoadBalancerIPs.md:14]: ✅

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md (hunk 2 above)
Comment (michael-cico) [reply content/en/docs/setup/customizing/ociLoadBalancerIPs.md:69]: ✅

**Pull request merged**